src/pages/myNotes/notes.jsx

```diff
@@ -1,49 +1,116 @@
-import React, { useState } from 'react';
+import React, { useEffect, useState } from 'react';
 
 import { ListItemText } from '@mui/material';
+import { ModalUnstyled } from '@mui/material';
 
 import ComponentInfo from './componentInfo';
 import NoteListItem from './noteListItem';
 import { todoList } from '../../constants/noteList';
-import { Container, NotesList, StyledList, StyledListComponent } from './styled';
+import { useLocalStorage } from '../../hooks/useLocaleStorage';
+import {
+    Backdrop,
+    ChangeDescription,
+    StyledInput,
+    Container,
+    EditButton,
+    ModalWindow,
+    NotesList,
+    StyledList,
+    StyledListComponent,
+} from './styled';
 
 const Notes = () => {
+    const [todosFromLocalStorage, setTodosFromLocalStorage] = useLocalStorage('todoList', todoList);
+
+    const [todos, setTodos] = useState([]);
     const [componentInfo, setComponentInfo] = useState({});
+    const [isEditModalOpen, setIsEditModalOpen] = useState(false);
+    const [changeDescriptionInputValue, setChangeDescriptionInputValue] = useState('');
+
+    useEffect(() => {
+        setTodos(todosFromLocalStorage);
+    }, []);
+
+    const setDescription = (element) => {
+        setChangeDescriptionInputValue(element.target.value);
+    };
+
+    const changeDescription = () => {
```

Comment:
* Rename it to "submitFormChanges", or something like this. If I will tell you to edit not only the description but the title as well?
* Try to make your functions as clean as possible, without dependencies. In this case, you can provide an edited element as a parameter and then you can do whatever you want with it, you don't need 'componentInfo' dependency. 
* Also when you want to edit 1 element from an array you can use .map and edit by condition